bootstrap/container/messaging.go

```diff
@@ -32,3 +33,18 @@ func MessagingClientFrom(get di.Get) messaging.MessageClient {
 
 	return client
 }
+
+type ExternalMessageClient interface {
+	messaging.MessageClient
```

Comment:
Needs to be a MQTT Client, not client from go-mod-messaging
```suggestion
	pahoMqtt.Client
```